### PR TITLE
Added GetProps

### PR DESCRIPTION
**Pull request by djgamerr**
Added GetProps(hash) as a way to get info about a specific torrent (trackers, seeders...etc)

### CODE REVIEW

File: UTorrentClientApi/Data/Props.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+namespace UTorrent.Api.Data
+{
+   public class Props
+    {
+        public string hash { get; set; }
```
Comment (latop2604) [inline UTorrentClientApi/Data/Props.cs]: Can you use Pascal case for property name?

Comment (djgamerr) [reply UTorrentClientApi/Data/Props.cs]: No problems, i am more used to camelCase

**Comment by djgamerr** [comment]
Should be fixed now

**Comment by latop2604** [comment]
Thx, but you forgot to update JsonParser.cs

Can you also remove the underscore from property names?

**Comment by djgamerr** [comment]
Done.
I have changed JsonParser before, but probably i forgot to commit it. I dont work with GitHub at all, I mostly use TFS

**Comment by latop2604** [comment]
Thx for your work. I will push some minor changes to Props model soon. next I will push a new version to nuget.org

**Comment by latop2604** [comment]
Fix view trackers feature from #11

**Comment by latop2604** [comment]
v1.1.2 pushed to nuget.org

**Comment by djgamerr** [comment]
No problems, If i add anything else i will make sure to create a new pull request.

**Pull request merged**